zebra/main.c

```diff
@@ -95,6 +98,7 @@ struct option longopts[] = {
 	{"label_socket", no_argument, NULL, 'l'},
 	{"retain", no_argument, NULL, 'r'},
 	{"vrfdefaultname", required_argument, NULL, 'o'},
+	{"kernel_reconcile", no_argument, NULL, 'K'},
```

Comment:
graceful restart and it should accept an argument of number of seconds to delay.  Default of 60 seconds

Reply:
+1 @donaldsharp is correct here, the default should be 60 seconds, not 15, and the delay should be configurable.

Reply:
Sure, I will make it configurable with -K option and keep default as 60 secs. Thanks.